pkg/background/generate/generate.go

```diff
@@ -91,6 +93,13 @@ func NewGenerateController(
 
 	c.statusControl = common.StatusControl{Client: kyvernoClient}
 
+	gvr, err := client.DiscoveryClient.GetGVRFromKind("Namespace")
```

Comment:
We should avoid using dynamic informers for performance concerns. Please create typed informer if needed.

Reply:
Right , there is other PR to take care of this , can we merge this and then it will be fixed as part of that PR.